sdk/ml/azure-ai-ml/azure/ai/ml/dsl/_group_decorator.py

```diff
@@ -188,9 +188,9 @@ def _create_fn(
         txt = f" def {name}({args}){return_annotation}:\n{body}"
         local_vars = ", ".join(locals.keys())
         txt = f"def __create_fn__({local_vars}):\n{txt}\n return {name}"
-        ns = {}
+        ns: Dict = {}
         exec(txt, globals, ns)  # pylint: disable=exec-used # nosec
-        return ns["__create_fn__"](**locals)
+        return cast(Callable, ns["__create_fn__"](**locals))
```

Comment:
See if there's another way to handle this without `cast`

Reply:
Fixed